python/lsst/ci/hsc/gen3/tests/mock.py

```diff
@@ -0,0 +1,71 @@
+# This file is part of ci_hsc_gen3.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Module to support mock option in unit tests.
+"""
+
+__all__ = ["check_mock", "skip_mock"]
+
+import unittest
+
+from lsst.daf.butler import Butler
+
+
+def check_mock(butler: Butler, dataset_type: str) -> bool:
```

Comment:
Is this used anywhere outside of this file?

Reply:
No, not yet at least. But it is a nice little function with a simple API that someone could use later (e.g. doing something different instead of just skipping the test).